mw-webapp/src/logic/userPage/UserPage.tsx

```diff
@@ -57,7 +57,7 @@ export const UserPage = (props: UserPageProps) => {
   const [userPreview, setUserPreview] = useState<UserPreview | null>(null);
   const navigate = useNavigate();
   const {user} = useUserContext();
-  const isOwner = user?.uid === userPreview?.uuid;
```

Comment:
PLease check do we need to use state with context for user. I think context is enough?

Reply:
We need use state and contex for UserPage.